### ISSUE TITLE
Metrics for the CNN -- f1_score

### ISSUE THREAD

**Issue by Aubert-Antoine** [Aubert-Antoine/EL-3003-IA-et-Deep-Learning#7]
```python 
def f1_m(y_true, y_pred):
    precision = precision_m(y_true, y_pred)
    recall = recall_m(y_true, y_pred)
    return 2*((precision*recall)/(precision+recall+K.epsilon()))

# compile the model
model.compile(optimizer='adam', loss='binary_crossentropy', metrics=['acc',f1_m,precision_m, recall_m])
```
*from this [site](https://datascience.stackexchange.com/questions/45165/how-to-get-accuracy-f1-precision-and-recall-for-a-keras-model)*

or : [https://www.tensorflow.org/addons/api_docs/python/tfa/metrics/F1Score](https://www.tensorflow.org/addons/api_docs/python/tfa/metrics/F1Score)

### Details on F1_score : 
[https://en.wikipedia.org/wiki/F-score](https://en.wikipedia.org/wiki/F-score)
